plugins/crane.yaml

```diff
@@ -0,0 +1,135 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: crane
+spec:
+  homepage: https://github.com/gocrane/kubectl-crane
+  shortDescription: "Easily interact with Crane"
+  description: |
+    Crane is a powerful FinOps platform that can analyze
+    the state of a kubernetes cluster and provide detailed
+    optimization recommendations. With these recommendations,
+    users can easily adjust their deployments to reduce costs
+    and increase efficiency. The kubectl plugin makes it easy
+    for users to follow these recommendations by allowing them
+    to manually adjust the number of replicas or requests for
+    a deployment with just a few simple commands.
+  caveats: |
```

Comment:
Let's delete caveats, we already show the user how to run the plugin which already prints this. (Plus this will go out of date.)

Reply:
Done.